arm-network/2016-09-01/swagger/network.json

```diff
@@ -8220,6 +8436,38 @@
       },
       "description": "Response for the ListVirtualNetworkGateways API service call."
     },
+    "BgpPeerStatusListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BgpPeerStatus"
+          },
+          "description": "Gets list of BGP peers"
```

Comment:
Just another quick reminder: No "Gets" and such. Note that this also affects descriptions that are not part of this commit, please fix them in the same run :)